### PR TITLE
[NH,CT,ID]: action classification updates

### PR DESCRIPTION
**Pull request by sroomf**
PR for action classification updates. These updates aim to increase action classification coverage as well as standardize the classification code so that it utilizes the BaseCategorizer class defined [here](https://github.com/openstates/openstates-scrapers/blob/3e5b5b1eec50f71a7b15b8be1c409a8be483694d/scrapers/utils/actions.py#L62).

### CODE REVIEW

File: scrapers/nh/actions.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    Rule(r"^(?!.*Committee Report).*[Rr]eferred [Tt]o.*$", "referral-committee"),
+    Rule(r"Lay.*on Table.*$", "deferral"),
+    Rule(
+        r"^(?!.*Minority).*Committee Report: Inexpedient to Legislate",
+        "committee-failure",
+    ),
+)
+
+
+class Categorizer(BaseCategorizer):
```
Comment (johnseekins) [inline scrapers/nh/actions.py:29]: I see this boilerplate in a few different scrapers. Is it possible to simplify this? Maybe make this part of the default object so people can just say `from utils.actions import Categorizer`? Or is that worth the effort?

**Comment by johnseekins** [comment]
`scrapers/id/actions.py` was committed as an empty file. Was this intentional?

File: scrapers/nh/actions.py (hunk 1 above)
Comment (sroomf) [reply scrapers/nh/actions.py:29]: Just for my understanding, are you suggesting we move this chunk of code:
```
class Categorizer(BaseCategorizer):
    rules = rules

    def categorize(self, text):
        attrs = BaseCategorizer.categorize(self, text)
        return attrs
```
to utils.actions so  we can directly import `Categorizer`? I like that idea but the `rules` definition would still need to happen in each jurisdiction's `actions.py`.  I think we would still need to have
```
class Categorizer(BaseCategorizer):
     rules = rules
```
in every actions.py?

**Comment by sroomf** [comment]
> `scrapers/id/actions.py` was committed as an empty file. Was this intentional?

File: scrapers/nh/actions.py (hunk 1 above)
Comment (johnseekins) [reply scrapers/nh/actions.py:29]: I missed that bit. Probably best to keep things this way and just document _how/why_ we import classifiers.

File: scrapers/id/bills.py [hunk 2]
```diff
@@ -294,7 +220,9 @@ def _split(string):
             if len(row[2]):
                 action = "".join(row[2].itertext())
             action = action.replace("\xa0", " ").strip()
-            atype = get_action(actor, action)
+            # atype = get_action(actor, action)
```
Comment (NewAgeAirbender) [inline scrapers/id/bills.py:223]: Can delete this instead of comment out

**Review by NewAgeAirbender — APPROVED** [review]
Looks good besides that one comment

File: scrapers/id/bills.py (hunk 2 above)
Comment (sroomf) [reply scrapers/id/bills.py:223]: good catch, thank you!

**Pull request merged**